src/cryptonote_core/service_node_list.cpp

```diff
@@ -1353,6 +1359,8 @@ namespace service_nodes
     }
     if ((args.size()-1)/ 2 > MAX_NUMBER_OF_CONTRIBUTORS)
     {
+      std::string msg = tr("Exceeds the maximum number of contributors, which is ") + MAX_NUMBER_OF_CONTRIBUTORS;
```

Comment:
This looks like it does pointer arithmetic on the string. 

```
 std::string msg = tr("Exceeds the maximum number of contributors, which is ") + std::to_string(MAX_NUMBER_OF_CONTRIBUTORS);
```

Reply:
Isn't `tr()` supposed to return `QString`?

Reply:
So it does then, and annoyingly it does not work the same way. I will add `to_string` in either case.